sdk/src/main/java/com/horizen/utils/MerklePathSerializer.java

```diff
@@ -1,15 +1,50 @@
 package com.horizen.utils;
 
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.JsonSerializer;
-import com.fasterxml.jackson.databind.SerializerProvider;
-import com.horizen.utils.BytesUtils;
+import scorex.core.serialization.ScorexSerializer;
+import scorex.util.serialization.Reader;
+import scorex.util.serialization.Writer;
+import java.util.ArrayList;
 
-import java.io.IOException;
+public class MerklePathSerializer<T extends MerklePath> implements ScorexSerializer<MerklePath> {
```

Comment:
`<T extends MerklePath>` can be removed, `ScorexSerializer` was already parameterized by `MerklePath`